docs/tutorials/README.md

```diff
@@ -0,0 +1,401 @@
+
+
+### Use the Cloud Console
+
+Connect to the Cloud Console and make sure your current project is selected. Then click on "Create" for a new VM instance:
+
+
+
+![alt_text](../images/image7.png )
+
+
+
+### Build a Container Optimized VM
+
+To create the VM use the steps in sequence below:
+
+
+
+*   Change the name of the VM to be something meaningful like "qsim-1".
+*   Choose a Region and Zone
+    *   The values are not too important now, but for latency and availability [you may make different choice](https://cloud.google.com/compute/docs/regions-zones#available).
+*   Choose the machine family / series: N2
+    *   Quantum simulation can require a high level of resources, so we are selecting powerful processors, but [many choices are available.](https://cloud.google.com/blog/products/compute/choose-the-right-google-compute-engine-machine-type-for-you)
+
+
+
+![alt_text](../images/image13.png )
+
+
+
+
+*   Choose the [Machine Type](https://cloud.google.com/blog/products/compute/choose-the-right-google-compute-engine-machine-type-for-you): n2-standard-16
+    *   16 CPUs
+    *   64GB memory
+*   Choose the Boot Disk image:[ Container-Optimized OS](https://cloud.google.com/container-optimized-os/docs/concepts/features-and-benefits)
+    *   Leave the remaining as defaults.
+
+
+
+
+![alt_text](../images/image10.png )
+
+
+
+
+Finally, enable HTTP access
+*   Click "Create"
+
+
+![alt_text](../images/image8.png )
+
+## Running the qsim docker container
+
+There are two ways to run qsim from this container: from a Jupyter Notebook and interactively from a shell. In both cases, you need to connect to the VM you created above.
+
+
+### Open a SSH window to your VM
+
+Once VM is created you can connect via [SSH](https://cloud.google.com/compute/docs/ssh-in-browser) to your newly created simulation VM. The easiest is to click the SSH button in the console from the **Compute Engine** -> **VM Instances** page:
+
+
+
+![alt_text](../images/image5.png )
+
+
+You will have a popup window connecting you to the simulation VM. You should have a command prompt:
+
+
+
+
+
+![alt_text](../images/image6.png )
+
+
+Start the container:
+
+
+```console
+    $ docker run -v `pwd`:/homedir -p 8888:8888 gcr.io/quantum-291919/jupyter_qsim
+```
+
+
+The output will be something like:
+
+
+```console
+    Unable to find image 'gcr.io/quantum-291919/jupyter_qsim:latest' locally
+    latest: Pulling from quantum-291919/jupyter_qsim
+    3c72a8ed6814: Pull complete 
+    2bd71a698eae: Pull complete 
+    Digest: sha256:d0d0040c6ef9925719459e736631c9fec957b94d377009c591d6285143ebb626
+    ...    
+        To access the notebook, open this file in a browser:
+            file:///root/.local/share/jupyter/runtime/nbserver-1-open.html
+        Or copy and paste one of these URLs:
+            http://79804d33f250:8888/?token=d7a53e728e3dff08c6ed12a15810471486b4c15828a3d70a
+         or http://127.0.0.1:8888/?token=d7a53e728e3dff08c6ed12a15810471486b4c15828a3d70a
+```
```

Comment:
```suggestion
```console
Unable to find image 'gcr.io/quantum-291919/jupyter_qsim:latest' locally
latest: Pulling from quantum-291919/jupyter_qsim
3c72a8ed6814: Pull complete 
2bd71a698eae: Pull complete 
Digest: sha256:d0d0040c6ef9925719459e736631c9fec957b94d377009c591d6285143ebb626
...    
    To access the notebook, open this file in a browser:
        file:///root/.local/share/jupyter/runtime/nbserver-1-open.html
    Or copy and paste one of these URLs:
        http://79804d33f250:8888/?token=d7a53e728e3dff08c6ed12a15810471486b4c15828a3d70a
     or http://127.0.0.1:8888/?token=d7a53e728e3dff08c6ed12a15810471486b4c15828a3d70a
```
```